<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BilateralDialog</class>
 <widget class="QDialog" name="BilateralDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bilateral Filter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="diameterLabel">
         <property name="text">
          <string>Diameter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="diameterSlider">
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>11</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="diameterSpinBox">
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>11</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="sigmaCLabel">
         <property name="text">
          <string>Sigma Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sigmaCSlider">
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="sliderPosition">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sigmaCSpinBox">
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="sigmaSLabel">
         <property name="text">
          <string>Sigma Space</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sigmaSSlider">
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sigmaSSpinBox">
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BilateralDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BilateralDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>diameterSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>diameterSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sigmaCSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sigmaCSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sigmaSSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sigmaSSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>diameterSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>diameterSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sigmaCSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sigmaCSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sigmaSSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sigmaSSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
